scripts/build-json-examples.ts

```diff
@@ -33,14 +33,14 @@ const walkDirectory = async (dir: string) => {
       // Write file to mirrored directory, outside the /data folder where the TS examples are stored
       const jsonExampleFilePath = path.join(
         dir.replace('/data', ''),
-        `${path.basename(file, '.ts')}.json`
+        `${path.basename(file, '.ts')}.json`,
       );
       await convertTypeScriptObjectToJSON(filePath, jsonExampleFilePath);
     }
   }
 };
 
-(async () => {
+void (async () => {
```

Comment:
The linter [was complaining about this][1] so i added `void` to keep it quiet

[1]: https://typescript-eslint.io/rules/no-floating-promises/#ignorevoid